code/modules/mob/living/silicon/robot/robot_model.dm

```diff
@@ -306,6 +315,17 @@
 	model_select_icon = "service"
 	cyborg_base_icon = "clown"
 	hat_offset = -2
+	//SKYRAT EDIT ADDITION
```

Comment:
Couldn't we keep all of these in a modular file, since it's just a variable attribution on the base definition.

Reply:
I suppose so yeh